core/src/com/unciv/models/ruleset/unique/UniqueTriggerActivation.kt

```diff
@@ -1010,7 +1045,7 @@ object UniqueTriggerActivation {
                 return {
                     unit.healBy(unique.params[1].toInt())
                     if (notification != null)
-                        unit.civ.addNotification(notification, MapUnitAction(unit), NotificationCategory.Units) // Do we have a heal icon?
+                        unit.civ.addNotification(notification, MapUnitAction(unit), NotificationCategory.Units, unit.name) // Do we have a heal icon?
```

Comment:
Should `unit.name` be `UnitIcons/${unit.name}` instead?

Reply:
I think I'll start a Notifications Kdoc branch. Also feels like _almost_ these icons need encapsulation - like make NotificationIcon an enum implementing an interface, have one or more RulesetObject-mapping implementations of the same interface, a string-to-one-of-those mapper, addNotification consumes a vararg of that interface,... Nah, serialization should stay strings...